index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="media-style.css">
+    <title>Anicet's Capstone Project</title>
+</head>
+
+<body>
+    <header>
+        <div class="logo"><a href=""><img src="./images/NFTSbck.jpg" alt="This a NFT logo"></a></div>
+
+        <ul>
+            <li><a href="About.html">About</a></li>
+            <li><a href="">Progran</a></li>
+            <li><a href="">Join</a></li>
+            <li><a href="">Sponsors</a></li>
+            <li><a href="">News</a></li>
+            <li><a href="">CC campaign</a></li>
+        </ul>
+    </header>
+    <main>
+        <div class="first-part">
+            <div class="nav-btn">
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+            </div>
+
+            <ul class="menu-bar">
+                <li class="close-button">&times;</li>
+                <li><a href="About.html">About</a></li>
+                <li>Progran</li>
+                <li>Join</li>
+                <li>Sponsors</li>
+                <li>News</li>
+                <li>CC campaign</li>
+            </ul>
+
+            <p class="punchline">"Hello cryptography world"</p>
+            <h1>Start the the day with coffee. <br> End the day with NFTS</h1>
+            <p class="punchline2">Anyone who follows the crypto space knows that non-fungible tokens (NFTs) are all the
+                rage of late.</p>
+            <div class="for-date">2021.05.11(THU) ~ 05(FRI)</div>
+            <div class="endline">@ National crypto of Democratic Republic of Congo</div>
+        </div>
+
+        <div class="for-cards">
+            <h2>Main Programs</h2>
+            <div class="card-container">
```

Comment:
- [ ] Kindly transform this `<div>` tag into a `<section>` tag for more semantic usage.
- [ ] This section should contain 5 article elements as seen in the original design. Please kindly add one more item.

![image](https://user-images.githubusercontent.com/4247740/158915551-decb1e19-6640-460b-b08d-9d3117a745c9.png)